ooniprobe/Base.lproj/Localizable.strings

```diff
@@ -105,3 +105,7 @@
 "view_data_policy" = "Click here to view our data policy";
 "test_name_finished" = "%@ finished, go to Past Tests to see the results";
 "open_url_alert" = "Do you want to open the following URL?";
+"max_runtime_low" = "The max runtime you have entered is too low, setting to default.";
+"clear_all_tests" = "Clear all tests";
+"clear_all_tests_alert" = "Do you want to remove all tests?";
+"past_tests_empty" = "Past tests restults will appear here once the tests have finished running";
```

Comment:
> "max_runtime_low" = "The max runtime you have entered is too low, setting to default.";

OK.

> "clear_all_tests" = "Clear all tests";

This should read:

"Delete all test results"

> "clear_all_tests_alert" = "Do you want to remove all tests?";

"Do you want to delete all test results"